static/templates/settings/unsubscribe-stream-modal.handlebars

```diff
@@ -0,0 +1,13 @@
+<div id="unsubscribe_stream_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="unsubscribe_stream_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+        <h3 id="unsubscribe_stream_modal_label">{{t "Unsubscribe to" }} <span class="stream_name">#{{stream_name}}</span></h3>
+    </div>
+    <div class="modal-body">
+        <p>{{t "You are the only user in this invite-only stream." }} <strong>{{t "Are you sure you want to do this?" }}</strong></p>
```

Comment:
We should also mention here that stream will automatically deactivated when last user unsubscribed from private stream.